src/server/chromium/crPage.ts

```diff
@@ -401,6 +401,22 @@ class FrameSession {
       const { windowId } = await this._client.send('Browser.getWindowForTarget');
       this._windowId = windowId;
     }
+
+    let screencastOptions: types.PageScreencastOptions | undefined;
+    if (this._isMainFrame() && this._crPage._browserContext._options.recordVideo && hasUIWindow) {
+      const size = this._crPage._browserContext._options.recordVideo.size || this._crPage._browserContext._options.viewport || { width: 1280, height: 720 };
```

Comment:
recordVideo.size should already be validated in `validateBrowserContextOptions`

Reply:
I'll address this in a follow up.